cmd/tracee-rules/input.go

```diff
@@ -53,6 +56,31 @@ func setupTraceeGobInputSource(opts *traceeInputOptions) (chan protocol.Event, e
 	gob.Register(map[string]trace.HookedSymbolData{})
 	gob.Register([]trace.DnsQueryData{})
 	gob.Register([]trace.DnsResponseData{})
+
+	// Network Protocol Event Types
+
+	// IPv4
+	gob.Register(trace.ProtoIPv4{})
```

Comment:
Note to self: I think we should move this gob register to some shared function we can keep between tracee-rules and tracee-ebpf instead of this double work with `printer.go`.